packages/parse5-parser-stream/lib/index.ts

```diff
@@ -42,10 +42,9 @@ export class ParserStream<T extends TreeAdapterTypeMap = DefaultTreeAdapterMap>
     constructor(options?: ParserOptions<T>) {
         super({ decodeStrings: false });
 
-        this.parser = new Parser(options);
-
-        this.document = this.parser.treeAdapter.createDocument();
-        this.parser._bootstrap(this.document, null);
+        const opts = { ...defaultParserOptions, ...options };
+        this.parser = new Parser(opts);
```

Comment:
Why aren’t defaults handled in `Parser`?

Reply:
Done!